src/BUILD

```diff
@@ -2712,6 +2715,7 @@ cc_test(
         "test/pipelinedefinitionstatus_test.cpp",
         "test/predict_validation_test.cpp",
         "test/prediction_service_test.cpp",
+        "test/pull_hf_model_test.cpp",
```

Comment:
separate test target. check eg openvino_tests/openvino_remote_tensor_tests.

Reply:
This is standard tests, does not require additional settings nor dependencies.